src/ScrollPager.tsx

```diff
@@ -188,6 +189,16 @@ export default class ScrollPager<T extends Route> extends React.Component<
           ref={this.scrollViewRef}
         >
           {children}
+          <Animated.Code
+            exec={onChange(
+              this.relativePosition,
+              cond(eq(round(this.relativePosition), this.relativePosition), [
```

Comment:
This has 0 tolerance for rounding errors or anything that might cause `this.relativePosition` to end up on a value that isn't an integer. Perhaps a better check would be to test whether `this.relativePosition` is very close to a round number. Using the logic from the following psuedo-code:

```
p = relativePosition
threshold = .005

atRest = p / round(p) <= (1 + threshold) && p / round(p) >= (1 - threshold)
```

```
and(
  lessOrEq(divide(this.relativePosition, round(this.relativePosition)), 1 + threshold),
  greaterOrEq(divide(this.relativePosition, round(this.relativePosition)), 1 - threshold)
)
``` 

/shrug, Thoughts?